content/api/openapi.yaml

```diff
@@ -574,6 +589,7 @@ paths:
       description: Fetches the ABI for a smart contract.
       operationId: accounts-abis
       parameters:
+        - $ref: '#/components/parameters/fmt'
```

Comment:
We'll have to see what effect adding this to every route does to the docs. I know you had to do this, and I suspect we will have to do this for other 'global' command-line options (such as `--file` or `--ether` and probably now `--chain`), but I'd rather if it didn't appear in the docs. We'll have to see how that works. Nothing to do here but note my comment.